src/data_source/storage/sql.rs

```diff
@@ -1051,6 +1072,11 @@ pub mod testing {
 
     impl TestMerkleTreeMigration {
         fn create(name: &str) -> String {
+            #[cfg(feature = "embedded-db")]
+            let bitvec_type = "TEXT";
+            #[cfg(not(feature = "embedded-db"))]
+            let bitvec_type = "BIT(8)";
```

Comment:
Might be nice to expose some of these publicly (maybe as constants in the `db` module?). I noticed we also have max function being either `MAX` or `GREATEST` in the sequencer. If we export all these little differences from here perhaps we can keep the `#[cfg]` gating limited to this crate

Reply:
exported it under syntax_helpers() module in db.rs